src/components/home/SectionFour.tsx

```diff
@@ -38,7 +49,7 @@ const SectionFour = () => {
             </div>
           </Card>
```

Comment:
![image](https://github.com/user-attachments/assets/d0775cf3-a45e-43bc-862a-3a70ff4fed3f)
navigator look prettier weird on responsive

Reply:
I resolved this by replacing the divide-y and divide-x with border-b and border-l so when you zoom or scroll it won't be look weird